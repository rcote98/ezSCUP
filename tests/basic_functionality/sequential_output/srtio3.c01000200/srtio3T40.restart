2	2	2
5	3
Sr	Ti	O
-3.00000000E-02	-3.00000000E-02	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	1	1	2.52187353E-02	1.06496938E-02	-3.28851002E-03
0	0	0	2	2	-9.30918956E-04	-1.31226465E-02	-2.13675588E-03
0	0	0	3	3	2.77990963E-01	2.81622603E-01	-1.11409471E-02
0	0	0	4	3	1.53093124E-01	-1.04129863E-02	-3.18018266E-01
0	0	0	5	3	-1.50200634E-02	-1.90344275E-01	-3.25667989E-01
0	0	1	1	1	-2.59293196E-02	-2.06545542E-02	4.51703101E-03
0	0	1	2	2	-4.28879586E-03	-7.32895391E-03	1.14647635E-02
0	0	1	3	3	-2.76415049E-01	-2.93802402E-01	1.11331939E-03
0	0	1	4	3	-3.04385724E-01	4.10151266E-03	3.17704739E-01
0	0	1	5	3	-6.52525730E-03	3.04043936E-01	3.01336973E-01
0	1	0	1	1	-2.73397114E-02	-4.48130697E-03	-2.45141891E-02
0	1	0	2	2	1.85742233E-02	-9.36016462E-03	-6.37794589E-04
0	1	0	3	3	-2.63914468E-01	-2.76848918E-01	1.04077401E-02
0	1	0	4	3	-1.69877827E-01	-1.24140875E-02	3.11569766E-01
0	1	0	5	3	5.33663722E-04	1.51453447E-01	3.19937618E-01
0	1	1	1	1	4.60378332E-02	-1.14061549E-02	5.74897869E-03
0	1	1	2	2	6.32930603E-03	2.96226087E-04	-4.75567199E-03
0	1	1	3	3	2.69692172E-01	2.93153493E-01	1.96078889E-02
0	1	1	4	3	3.00550768E-01	-2.75319780E-03	-3.10665394E-01
0	1	1	5	3	2.55530249E-03	-3.19518562E-01	-2.55384133E-01
1	0	0	1	1	-1.66271436E-02	-1.49314867E-02	-1.01459218E-02
1	0	0	2	2	-6.03812860E-03	1.68823625E-02	-2.71010560E-02
1	0	0	3	3	-2.87689001E-01	-2.41589066E-01	-1.61336012E-02
1	0	0	4	3	-1.87432071E-01	-5.43974459E-04	2.80831604E-01
1	0	0	5	3	-3.69448540E-03	1.47144191E-01	3.12931281E-01
1	0	1	1	1	3.79536510E-02	2.96563540E-02	1.28581718E-03
1	0	1	2	2	-7.92531113E-04	2.32213008E-02	-5.94759724E-03
1	0	1	3	3	2.83572078E-01	2.68455977E-01	-2.24526733E-04
1	0	1	4	3	3.06967856E-01	8.15987077E-03	-2.84828534E-01
1	0	1	5	3	1.46179639E-03	-3.09116642E-01	-2.84933668E-01
1	1	0	1	1	2.08040712E-02	1.02266662E-02	-1.11178121E-02
1	1	0	2	2	1.74530706E-02	1.45818212E-02	8.55168673E-03
1	1	0	3	3	2.68833353E-01	2.72053513E-01	-2.17514074E-03
1	1	0	4	3	1.36415053E-01	3.39327178E-03	-3.09385799E-01
1	1	0	5	3	3.23195706E-03	-1.71833370E-01	-2.64503372E-01
1	1	1	1	1	-2.36460311E-02	-2.38314853E-02	-1.34541850E-03
1	1	1	2	2	7.00573307E-03	2.69956295E-02	-4.49911644E-03
1	1	1	3	3	-2.66734001E-01	-2.63444050E-01	4.59594767E-03
1	1	1	4	3	-2.98615431E-01	9.97573537E-03	2.82227863E-01
1	1	1	5	3	1.62125100E-03	3.21670682E-01	2.84718191E-01
