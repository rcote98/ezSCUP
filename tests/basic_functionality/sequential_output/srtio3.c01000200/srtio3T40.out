
*****************************************************************
*                          Welcome to                           *
*                           SCALE-UP                            *
*                                                               *
*            Second-Principles Computational Approach           *
*                             for                               *
*                     Lattice and Electrons                     *
*                                                               *
*                       Version:         1.0.0.Gorbea           *
*                    Build date: 20200801T181                   *
*****************************************************************
*                    SCALE-UP is property of                    *
*              the University of Cantabria (Spain)              *
*    and the Luxembourg Institute of Science and Technology     *
*****************************************************************
* Authors:                                                      *
*                                                               *
*  - Pablo Garcia-Fernandez   garciapa@unican.es                *
*  - Jorge Iñiguez            jorge.iniguez@list.lu             *
*  - Javier Junquera          junqueraj@unican.es               *
*****************************************************************
* SCALE-UP is based on two earlier programs:                    *
*                                                               *
* -  SPLD ('Second-Principles Lattice Dynamics'), developed     *
*    at the ICMAB-CSIC (Spain) by J.C. Wojdel, M.P. Ljungberg   *
*    and J. Iñiguez, and distributed under MIT license.         *
*                                                               *
* -  MKII a second-principles electron code                     *
*    developed at the University of Cantabria by                *
*    P. Garcia-Fernandez  and J. Junquera.                      *
*****************************************************************
*                                                               *
* Bibliography:                                                 *
*                                                               *
*  P. Garcia-Fernandez et al. Phys. Rev. B 93, 195137 (2016)    *
*  J. C. Wojdel et al., J. Phys. Cond. Matt. 25, 305401 (2013)  *
*****************************************************************
Reading parameter file... lattice... done!
Reading restart file     srtio3T40.restart                                                               

Basic data:
  Number of CPUs:               1
  Run mode:            Montecarlo
  Lattice:                 Active
  Electrons:           NOT Active
  External electric field, static. Field components (V/m):   0.000000E+00   0.000000E+00   0.000000E+00

Reference system:
 Supercell data:
  Lattice constants (Ang):   7.690   7.690   7.690
  Lattice angles (degree):  90.000  90.000  90.000
 Supercell size:   2   2   2

 Total number of atoms          :      40
 Total number of orbitals       :       0
 Total number of active orbitals:       0

 Atom in unit cell:
 Atom   Label       Mass  Charge 
    1   Sr         87.60     0.0
    2   Ti         47.90     0.0
    3   O          16.00     0.0
    4   O          16.00     0.0
    5   O          16.00     0.0

 Orbitals in unit cell:
 Orbital   Label  

 Background dielectric tensor:
    6.2319     0.0000     0.0000
    0.0000     6.2319    -0.0000
    0.0000    -0.0000     6.2319

Initial values (basic output MC simulation)
Energy (eV):            -0.298475E+05
Polarization (C/m2):     0.011388    0.007407   -0.008635
Strain:                 -0.030000   -0.030000    0.000000    0.000000    0.000000    0.000000

LT:    Iter       Temp(K)    Energy(eV)   Pol_x(C/m2)   Pol_y(C/m2)   Pol_z(C/m2) Av_Pl_x(C/m2) Av_Pl_y(C/m2) Av_Pl_z(C/m2)       Strn_xx       Strn_yy       Strn_zz       Strn_yz       Strn_xz       Strn_xy    Av_Strn_xx    Av_Strn_yy    Av_Strn_zz    Av_Strn_yz    Av_Strn_xz    Av_Strn_xy 
LT:      10  0.400000E+02 -0.298474E+05  0.240898E-01  0.265110E-01 -0.421649E-01  0.444058E-02  0.238806E-01 -0.209121E-01 -0.300000E-01 -0.300000E-01  0.246330E-02 -0.178591E-02 -0.421595E-03  0.000000E+00 -0.330000E-01 -0.330000E-01  0.169212E-02 -0.960510E-03 -0.548930E-04  0.000000E+00 
LT:      20  0.400000E+02 -0.298475E+05  0.375842E-02  0.219745E-01 -0.885576E-01  0.361045E-02  0.257340E-01 -0.368296E-01 -0.300000E-01 -0.300000E-01  0.485828E-02 -0.416603E-02 -0.387626E-03  0.000000E+00 -0.315000E-01 -0.315000E-01  0.281621E-02 -0.213133E-02 -0.188788E-03  0.000000E+00 
LT:      30  0.400000E+02 -0.298476E+05  0.221354E-01  0.186833E-01 -0.222149E-01  0.512003E-02  0.235749E-01 -0.405237E-01 -0.300000E-01 -0.300000E-01  0.941440E-02 -0.339035E-02 -0.151432E-02  0.000000E+00 -0.310000E-01 -0.310000E-01  0.421318E-02 -0.221539E-02 -0.549615E-03  0.000000E+00 
LT:      40  0.400000E+02 -0.298476E+05  0.412288E-01 -0.625759E-02  0.539890E-02  0.112629E-01  0.205881E-01 -0.303116E-01 -0.300000E-01 -0.300000E-01  0.132310E-01 -0.656618E-02 -0.394858E-02  0.000000E+00 -0.307500E-01 -0.307500E-01  0.635653E-02 -0.299898E-02 -0.117396E-02  0.000000E+00 
LT:      50  0.400000E+02 -0.298477E+05 -0.334739E-02 -0.273464E-01  0.330938E-01  0.990803E-02  0.113021E-01 -0.209114E-01 -0.300000E-01 -0.300000E-01  0.163189E-01 -0.694331E-02 -0.280124E-02  0.000000E+00 -0.306000E-01 -0.306000E-01  0.820470E-02 -0.376946E-02 -0.164880E-02  0.000000E+00 
LT:      60  0.400000E+02 -0.298477E+05  0.300503E-02 -0.285439E-01  0.327446E-01  0.100142E-01  0.642885E-02 -0.123907E-01 -0.300000E-01 -0.300000E-01  0.199323E-01 -0.269009E-02 -0.449500E-02  0.000000E+00 -0.305000E-01 -0.305000E-01  0.100076E-01 -0.381644E-02 -0.184816E-02  0.000000E+00 
LT:      70  0.400000E+02 -0.298478E+05 -0.211725E-02  0.111225E-01 -0.273170E-01  0.969624E-02  0.511083E-02 -0.895590E-02 -0.300000E-01 -0.300000E-01  0.199495E-01 -0.350280E-02 -0.118951E-02  0.000000E+00 -0.304286E-01 -0.304286E-01  0.114224E-01 -0.350944E-02 -0.199796E-02  0.000000E+00 
LT:      80  0.400000E+02 -0.298479E+05 -0.186117E-02 -0.134674E-01  0.335616E-01  0.989062E-02  0.432199E-02 -0.779027E-02 -0.300000E-01 -0.300000E-01  0.185013E-01 -0.626190E-02  0.146508E-02  0.000000E+00 -0.303750E-01 -0.303750E-01  0.124793E-01 -0.381267E-02 -0.171556E-02  0.000000E+00 
LT:      90  0.400000E+02 -0.298479E+05  0.536548E-03 -0.148702E-01  0.564181E-01  0.817204E-02  0.183359E-02 -0.101849E-02 -0.300000E-01 -0.300000E-01  0.207750E-01 -0.887185E-02 -0.566922E-03  0.000000E+00 -0.303333E-01 -0.303333E-01  0.133118E-01 -0.426839E-02 -0.141910E-02  0.000000E+00 
LT:     100  0.400000E+02 -0.298479E+05 -0.224581E-02  0.234799E-01 -0.268330E-01  0.728574E-02  0.139694E-02  0.121535E-02 -0.300000E-01 -0.300000E-01  0.213562E-01 -0.102555E-01 -0.338199E-03  0.000000E+00 -0.303000E-01 -0.303000E-01  0.141142E-01 -0.489357E-02 -0.135761E-02  0.000000E+00 

Final values (basic output MC simulation)
Energy (eV):            -0.298479E+05
Polarization (C/m2):    -0.002246    0.023480   -0.026833
Strain:                 -0.030000   -0.030000    0.021356   -0.010256   -0.000338    0.000000

-----------------------------------------------------------------
 Backing up the input read by fdf:

System_name     srtio3T40
Print_routineinfo         5     # default value
Print_routinedepth         0     # default value
Print_inout         0     # default value
Print_interactions         0     # default value
Print_k-vectors         0     # default value
Print_energies         1     # default value
Print_wavefunctions        -1     # default value
Print_electrostatic         1     # default value
Print_orbocc         1     # default value
Print_bands        -1     # default value
Print_forces        -1     # default value
Print_detail        -1     # default value
Print_Just_Geometry        -1     # default value
Print_Velocity         0     # default value
Print_Space_Hamiltonian        -1     # default value
Print_space_denmat        -1     # default value
Print_Geometry         1     # default value
Print_geoformat     normal     # default value
Print_test_file        -1     # default value
Print_local_lattice_energy        -1     # default value
Print_local_current        -1     # default value
Print_std_electron_nsteps         1     # default value
Print_std_lattice_nsteps             10
Print_std_energy              T
Print_std_polarization              T
Print_std_current              F     # default value
Print_std_strain              T
Print_std_av_polarization              T
Print_std_av_strain              T
Print_std_av_energy              F     # default value
Print_std_delta_energy              F     # default value
Print_std_temperature              T     # default value
Print_std_maximum_force              F     # default value
Print_std_maximum_stress              F     # default value
Print_std_mdd              T     # default value
Print_std_fermi              T     # default value
Print_std_pulay              T     # default value
Print_std_neh              T     # default value
Print_std_el_pol              T     # default value
Print_std_micro              F     # default value
run_mode     monte_carlo
Print_std_mdd              F     # default value
Print_std_fermi              F     # default value
Print_std_pulay              F     # default value
Print_std_neh              F     # default value
Print_std_electron_nsteps     10000     # default value
No_electron
%block fix_strain_component
 T T F F F T
T                                                                                             T
T                                                                                             T
F                                                                                             F
F                                                                                             F
F                                                                                             F
T                                                                                             T
Parameter_file     srtio3_full_lat.xml
%block Supercell
 2 2 2
%endblock Supercell
electric_field_scale          1.000000000     v/m     # default value
%block static_electric_field
 0.0 0.0 0.0
temperature         0.1000000000E-98 kelvin     # default value
%block external_stress
 0.0 0.0 0.0 0.0 0.0 0.0
Charge_tolerance         0.1000000000E-07     # default value
Geometry_restart
Geometry_restart     srtio3T40.restart
MC_nsweeps
MC_nsweeps            100
MC_temperature
MC_temperature          40.00000000         kelvin
# above item originally: MC_temperature          40.00000000     kelvin
MC_annealing_rate          1.000000000
MC_init_step_d         0.9448629925E-01 bohr     # default value
MC_init_step_s         0.1889725985E-02 bohr     # default value
MC_max_step_d         0.9448634389           bohr
# above item originally: MC_max_step_d         0.5000000000     ang
MC_min_step_d         0.1889725985E-05 bohr     # default value
MC_target_acc_d         0.5000000000         # default value
MC_target_acc_s         0.5000000000         # default value
MC_strains              T
n_write_mc
n_write_mc             10
Print_std_lattice_nsteps
*****************************************************************
 Memory usage:
  Function             Total_memory(Mb)  Max_memory(Mb)      Node
  setup_main                     0.0256          0.0256         0
  setup_basicvariables           0.0056          0.0056         0
  setup_latt_hamil               0.0014          0.0014         0
  setup_init_data_node           0.0256          0.0256         0
  hel_setup_structfactors        0.0256          0.0256         0
  hlt_create_supercell           0.0014          0.0014         0

 Timing statistics (seconds)
  Function                       Wall   Max_Wall    Max_CPU  Node
  setup_main                     3.73       3.73       3.72     0
  setup_readinput                3.50       3.50       3.50     0
  montecarlo_main                0.21       0.21       0.21     0
  montecarlo_sweep               0.20       0.20       0.20     0
  io_readparameterfile           3.50       3.50       3.50     0
  setup_latt_hamil               0.22       0.22       0.22     0
  hlt_add_harmonic_didi          0.22       0.22       0.22     0

 Total memory usage:     0.03 MB
 Total WALL:           3.937 seconds
*****************************************************************
*                                                               *
*               Succesfully ending SCALE-UP                     *
*                                                               *
*****************************************************************
